docs/content/topics/project-developers/projects.md

```diff
@@ -9,296 +9,267 @@ aliases:
-- *Upload a default brigade.json config*: The local path to a default brigade.json config file that will be used if none exists in the repo. Overrides the ConfigMap config
-
-#### Storing a Script in a ConfigMap
-
-It is possible to store a `brigade.js` script in a dedicated ConfigMap, and then share that ConfigMap with multiple projects.
-
-For example, if you have a local file named `examples/brigade.js`, you can use the `kubectl` command to turn it into a ConfigMap for Brigade to consume:
+Before we can create a Brigade project, we need to define the project itself.
+We will discuss the project definition file and then explore how to create and
+manage a project from on a given definition file.  If you'd like to skip ahead
+and learn how to streamline project creation via the `brig init` command,
+proceed to the [brig init section] below.
 
-```console
-$ kubectl create configmap default-brigade-script --from-file=brigade.js=examples/brigade.js
-```
+[brig init section]: #brig-init
 
-The command above stores the `examples/brigade.js` script in a ConfigMap named `default-brigade-script`. When creating a new project, you can answer the *Default script ConfigMap name* question by providing the name of this ConfigMap: `default-brigade-script`.
+### Project definition files
 
-If this value is supplied, the `brigade.js` in the ConfigMap will be used if and only if neither the project nor the repository provides an alternative `brigade.js` file. In other words, this is the _last location_ checked for a Brigade script.
+Brigade project definition files are represented in YAML or JSON and follow a
+schema that will look familiar to users who have dealt with Kubernetes
+manifests - however, they are their own entities and are only understood by
+Brigade and the brig CLI.
 
-This same pattern can be used to store a default `brigade.json` configuration file in a ConfigMap for use by multiple projects. The associated prompt is *Default brigade.json config ConfigMap name*
+This approach allows users of Brigade to persist project configuration in a VCS
+of their choice.  Updating a pre-existing project is as easy as supplying the
+updated configuration to the corresponding brig command.
 
-#### Storing a Default Script in the Project
+As an example, let's look at a project definition expressed in YAML and break
+it down into its main sections:
 
-It is also possible to store a default Brigade script within the project definition. When `brig project create` prompts for *Upload a default brigade.js script* you can enter the local path to a `brigade.js` file (e.g. `examples/brigade.js`) and that file will be copied into the project. Note that if you edit the `brigade.js` you will need to edit the project's Kubernetes Secret and update the script accordingly.
+```yaml
+# yaml-language-server: $schema=https://raw.githubusercontent.com/brigadecore/brigade/v2/v2/apiserver/schemas/project.json
```

Comment:
I'm wondering if we should take this line out. It's VS Code specific. Apart from that, I think this URL will change to something nice when we go GA.

Your call. I'm just thinking out loud.